### PR TITLE
Added support for --name, --adapter and --template CLI arguments for unattended Keystone app creation

### PR DESCRIPTION
**Pull request by gabipetrovay**
Enhanced the create-keastone-app CLI functionality:

* added support for `--name` and `--adapter` CLI arguments
* fixed the unused `--template` CLI argument

The user can now use the following command to create a Keystone application without any prompts (unattended installation).

```bash
create-keystone-app --name "My App" --template "starter" --adapter "Mongoose" my-app
```

or using npm:

```bash
npm init keystone-app --name "My App" --template "starter" --adapter "Mongoose" my-app
```

There are issues with `yarn create` because it does not pass the arguments to create-keystone-app:

```bash
# NOT WORKING
yarn create keystone-app --name "My App" --template "starter" --adapter "Mongoose" my-app
```

### CODE REVIEW

File: packages/create-keystone-app/lib/get-example-project.js [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
   if (EXAMPLE_PROJECT_CHOICE) {
     return EXAMPLE_PROJECT_CHOICE;
   }
 
+  // If the project template was provided via the CLI arguments
+  const args = getArgs();
+  const argValue = args['--template'];
+  if (argValue) {
+    projects.map(project => {
+      if (project.folder == argValue) {
```
Comment (jesstelford) [inline packages/create-keystone-app/lib/get-example-project.js]: It's a good idea to prefer strict equality here: `===` to ensure consistency with the rest of the codebase.

Comment (gabipetrovay) [reply packages/create-keystone-app/lib/get-example-project.js]: Fixed in 42d96669

**Comment by jesstelford** [comment]
@all-contributors can you add @gabipetrovay for code please? <3

**Comment by gabipetrovay** [comment]
> @jesstelford
> 
> I've put up [a pull request](https://github.com/keystonejs/keystone/pull/2275) to add @gabipetrovay! 🎉

❤️ Thanks!

**Comment by jesstelford** [comment]
@gabipetrovay If you're up for it, these new options could also use some documentation: https://github.com/keystonejs/keystone/issues/2276

**Pull request merged**